scripts/copy_dataset.py

```diff
@@ -0,0 +1,72 @@
+""" Copy JSON data files from a source directory to a destination directory with optional compression
+Usage:
+- with compression:    `python copy_dataset path/to/src/dir path/to/dst/dir 1`
+- without no compression: `python copy_dataset path/to/src/dir path/to/dst/dir 0`
+"""
+
+import sys
+import os
+import json
+
+import brotli
+
+def copy_dataset(src_dir: str, dst_dir: str, compress: bool = False)-> int:
+    """Copy JSON data files from a source directory to a destination directory with optional compression"""
+    print("----------------")
+    print("Copy dataset:")
+    print(f"Dest dir: {dst_dir}")
+    print(f"compress: {compress}")
+    print("----------------")
+
+    if not os.path.isdir(src_dir):
+        raise FileNotFoundError(f"source directory not found at '{src_dir}'")
+    if not os.path.isdir(dst_dir):
+        os.makedirs(dst_dir)
+
+    for i in os.scandir(src_dir):
+        if i.is_file():
+            src_path = os.path.join(src_dir, i.name)
+            dst_path = os.path.join(dst_dir, i.name)
+
+            decompressed_dict = None
+
+            with open(src_path, "rb") as compr_json_file:
+                compr_bytes = compr_json_file.read()
+                try:
+                    decompr_bytes = brotli.decompress(compr_bytes)
+                    decompressed_dict = json.loads(decompr_bytes.decode("utf-8"))
+                except brotli.error:
+                    try:
+                        decompressed_dict = json.loads(compr_bytes.decode("utf-8"))
+                    except json.JSONDecodeError:
+                        print(f"Invalid JSON file: {i.name}. Ignored.")
+
+            if decompressed_dict is None:
+                continue
+
+            json_str = json.dumps(decompressed_dict, separators=(',', ':'))
+
+            # Issue #527: replace `id` by seq_id in tasks entities and communication nodes
```

Comment:
Why is this commented out? If not needed it should be removed.

Reply:
Lines have been removed